presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueriesWithExchangeMaterialization.java

```diff
@@ -38,6 +40,13 @@ public void testMaterializedExchangesEnabled()
         assertQuery(getSession(), "SELECT orderkey, COUNT(*) lines FROM lineitem GROUP BY orderkey", assertRemoteMaterializedExchangesCount(1));
     }
 
+    @Test
+    public void testMaterializedExchangesWithLimitedConcurrency()
```

Comment:
I wonder it it makes sense to extract the `materializeExchangesSession` into a parameter in the  `TestHiveIntegrationSmokeTest#testMaterializedPartitioning`, and run it with `max_concurrent_materializations=1`, `max_concurrent_materializations=2` and the default as well.

Reply:
done